### PR TITLE
Include Brazil Flag

### PR DESCRIPTION
**Pull request by esganzerla**

### CODE REVIEW

File: Flags/Brazil.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Brazil | Flags</title>
+    <style> #brazil {
+        width: 320px;
```
Comment (ManrajGrover) [inline Flags/Brazil.html]: Please move `#brazil` to next line.

File: Flags/Brazil.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        height: 225px;
+        background-color: #FFE11F;
+        background-image: 
+			linear-gradient(
```
Comment (ManrajGrover) [inline Flags/Brazil.html]: Nit: Please fix indentation.

File: Flags/Brazil.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+				#16B83E 70%
+			);
+        box-shadow: 0px 0px 15px 2px #e5e5e5;
+        position: relative;
+      }
+      #brazil::before {
+        content: "";
+        position: absolute;
+        display: block;
+		width: 110px;
```
Comment (ManrajGrover) [inline Flags/Brazil.html]: Nit: Please fix indentation.

**Comment by ManrajGrover** [comment]
@esganzerla Thank you for contributing. :smile: I have left few fixes required. Apart from that, please update the README.md as per [Contributions](https://github.com/ManrajGrover/SingleDivProject/blob/master/CONTRIBUTIONS.md) guidelines and also add snapshot of the flag.

**Comment by esganzerla** [comment]
@ManrajGrover I've fixed those issues. In fact I worked a little more to make to organize better the spacings. Thanks

**Comment by ManrajGrover** [comment]
@esganzerla This differs a little when compared to original flag. Can you share some sources where one developed by you is official? Brilliant work btw. :smile:

**Comment by esganzerla** [comment]
@ManrajGrover You mean like this?

**Comment by ManrajGrover** [comment]
@esganzerla Yeps

**Comment by esganzerla** [comment]
Hey @ManrajGrover, I actually managed to insert the stars to the flag. Should do anything else to close this pull request?

**Comment by ManrajGrover** [comment]
@esganzerla Brilliant work! :smile: This looks perfect to me. One thing left is to resolve the merge conflicts.

**Comment by esganzerla** [comment]
Seems to me that your are the one who can merge, isn't it?

I have experience with pull requests, can you help me if I have to do it mysel?

Thanks

**Comment by ManrajGrover** [comment]
@esganzerla Thank you for fixing the conflicts! 😄

**Pull request merged**